engines/bastion_katello/test/products/details/product-repositories.controller.test.js

```diff
@@ -1,5 +1,5 @@
 describe('Controller: ProductRepositoriesController', function() {
-    var $scope, $q, expectedTable, expectedIds, Repository, RepositoryBulkAction, Nutupane;
+    var $scope, $q, expectedTable, expectedIds, Repository, RepositoryBulkAction, Nutupane, DownloadPolicy;
 
     beforeEach(module('Bastion.products', 'Bastion.test-mocks'))
```

Comment:
Including Bastion.repositories in this list here will fix this since that is the module that includes the DownloadPolicy service.